puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/deploy_yum.yaml

```diff
@@ -0,0 +1,33 @@
+- job:
+    name: deploy_yum
+    project-type: workflow
+    logrotate:
+      daysToKeep: 2
+      numToKeep: -1
+    triggers:
+      - timed: "" # ???
```

Comment:
I don't actually think you need triggers at all if you have nothing use with it.

Reply:
removed.